shared/src/search/parser/parser.ts

```diff
@@ -233,7 +233,7 @@ const whitespace = pattern(/\s+/, { type: 'whitespace' as const }, 'whitespace')
 
 const literal = pattern(/[^\s]+/)
 
-const filterKeyword = pattern(/-?[a-z]+(?=:)/)
+const filterKeyword = pattern(/-?[a-zA-z]+(?=:)/)
```

Comment:
You probably meant `A-Z`?

```suggestion
const filterKeyword = pattern(/-?[a-zA-Z]+(?=:)/)
```

Reply:
Yeah, this PR actually makes the case insensitivity changes: https://github.com/sourcegraph/sourcegraph/pull/7932 and should be merged first.